src/core/labeling/qgslabelfeature.h

```diff
@@ -299,15 +299,53 @@ class CORE_EXPORT QgsLabelFeature
     /**
      * Applies to "around point" placement strategy or linestring features.
      * Distance of the label from the feature (in map units)
+     *
+     * \see setDistLabel()
+     * \see maximumDistance()
      */
     double distLabel() const { return mDistLabel; }
 
     /**
      * Applies to "around point" placement strategy or linestring features.
      * Set distance of the label from the feature (in map units)
+     *
+     * \see distLabel()
+     * \see setMaximumDistance()
      */
     void setDistLabel( double dist ) { mDistLabel = dist; }
 
+    /**
+     * Returns the maximum distance which labels are allowed to be from their corresponding points.
+     *
+     * This setting works alongside distLabel() to define a permissible
+     * range of distances at which labels can be placed from their points.
+     *
+     * The default value is 0, which indicates that no maximum is set and the that distLabel()
+     * always be respected.
+     *
+     * \see setMaximumDistance()
+     * \see distLabel()
```

Comment:
Completely unrelated to your PR: TODO QGIS 4.0: distLabel() -> distanceToLabel() or something :laughing: